cmd/serve.go

```diff
@@ -50,7 +47,11 @@ DON'T DO THIS IN PRODUCTION!
 `)
 		}
 
-		watchAndValidateViper()
+		schema, err := schemas.Find("config.schema.json")
```

Comment:
Please move this back to `watchAndValidateViper` to reduce duplication in cases where we need to load the schema again